data/utils/defectsFromBias.py

```diff
@@ -112,13 +46,20 @@ def writeDefectsFile(bboxList, path, detectorSerial, detectorName):
     args = parser.parse_args()
 
     biasMI = afwImage.MaskedImageF(args.bias)
-    defectList = getDefectListFromMask(biasMI, "BAD")
-    bboxList = [defect.getBBox() for defect in defectList]
-    writeDefectsFile(bboxList, DefectsPath, detectorSerial, detectorName)
+    defectList = Defects.fromMask(biasMI, "BAD")
+    valid_start = date_parser.parse('19700101T000000')
+    md = defectList.getMetadata()
+    md['INSTRUME'] = 'test'
+    md['DETECTOR'] = detectorName
+    md['CALIBDATE'] = valid_start.isoformat()
+    md['FILTER'] = None
+    md['CALIB_ID'] = (f'detector={detectorName} calibDate={valid_start.isoformat()} '
+                      'ccd={detectorName} ccdnum={detectorName} filter=None')
```

Comment:
You need an f on the second string! For example (replacing the parens, which I personally find confusing, with a line continuation character):
```
md['CALIB_ID'] = f'detector={detectorName} calibDate={valid_start.isoformat()} ' \
                 f'ccd={detectorName} ccdnum={detectorName} filter=None'
```

The fact that this slipped through suggests that an additional test is needed.

Reply:
This is actually no longer used.  I've removed it elsewhere, but missed it here.